internal/testdata/complex/docker-compose.resources.yml

```diff
@@ -0,0 +1,105 @@
+    - ./docroot/sites/default/files:/app/docroot/sites/default/files:${VOLUME_FLAGS:-delegated} ### Local overrides to mount host filesystem. Automatically removed in CI and PROD.
+
+x-environment:
+  &default-environment
+  LAGOON_PROJECT: *project
+  DRUPAL_HASH_SALT: fakehashsaltfakehashsaltfakehashsalt
+  LAGOON_LOCALDEV_URL: ${LOCALDEV_URL:-http://mysite.docker.amazee.io}
+  LAGOON_ROUTE: ${LOCALDEV_URL:-http://mysite.docker.amazee.io}
+  GITHUB_TOKEN: ${GITHUB_TOKEN:-}
+  EXAMPLE_KEY: ${EXAMPLE_KEY:-}
+  EXAMPLE_IMAGE_VERSION: ${EXAMPLE_IMAGE_VERSION:-latest}
+  LAGOON_ENVIRONMENT_TYPE: ${LAGOON_ENVIRONMENT_TYPE:-local}
+  DRUPAL_REFRESH_SEARCHAPI: ${DRUPAL_REFRESH_SEARCHAPI:-}
+  EXAMPLE_INGRESS_PSK: ${EXAMPLE_INGRESS_PSK:-}
+  EXAMPLE_INGRESS_HEADER: ${EXAMPLE_INGRESS_HEADER:-}
+  EXAMPLE_INGRESS_ENABLED: ${EXAMPLE_INGRESS_ENABLED:-}
+  DB_ALIAS: ${DB_ALIAS:-bay.production}
+
+
+services:
+
+  cli:
+    build:
+      context: internal/testdata/complex/docker
+      dockerfile: .docker/Dockerfile.cli
+      args:
+        COMPOSER: ${COMPOSER:-composer.json}
+        EXAMPLE_IMAGE_VERSION: *example-image-version
+    image: *project
+    environment:
+      << : *default-environment
+    << : *default-volumes
+    volumes_from: ### Local overrides to mount host SSH keys. Automatically removed in CI.
+      - container:amazeeio-ssh-agent ### Local overrides to mount host SSH keys. Automatically removed in CI.
+    labels:
+      lagoon.type: cli-persistent
+      lagoon.persistent: /app/docroot/sites/default/files/
+      lagoon.persistent.name: nginx-php
+      lagoon.persistent.size: 5Gi
+
+  nginx:
+    build:
+      context: internal/testdata/complex/docker
+      dockerfile: .docker/Dockerfile.nginx-drupal
+      args:
+        CLI_IMAGE: *project
+        EXAMPLE_IMAGE_VERSION: *example-image-version
+    << : *default-volumes
+    environment:
+      << : *default-environment
+    depends_on:
+      - cli
+    networks:
+      - amazeeio-network
+      - default
+    labels:
+      lagoon.type: nginx-php-persistent
+      lagoon.persistent: /app/docroot/sites/default/files/
+      lagoon.persistent.size: 5Gi
+      lagoon.name: nginx-php
+      # NGINX_CPU_REQUEST is injected as project env in test
+      lagoon.resources.requests.cpu: ${NGINX_CPU_REQUEST}
+      lagoon.resources.limits.cpu: 100m
+    expose:
+      - "8080"
+  php:
+    build:
+      context: internal/testdata/complex/docker
+      dockerfile: .docker/Dockerfile.php
+      args:
+        CLI_IMAGE: *project
+        EXAMPLE_IMAGE_VERSION: *example-image-version
+    environment:
+      << : *default-environment
+    << : *default-volumes
+    depends_on:
+      - cli
+    labels:
+      lagoon.type: nginx-php-persistent
+      lagoon.persistent: /app/docroot/sites/default/files/
+      lagoon.persistent.size: 5Gi
+      lagoon.name: nginx-php
+      lagon.resources.requests.cpu: 20m
```

Comment:
Is this intentionally `lagon` instead of `lagoon`?

Reply:
Nope heh - I noticed a failing test locally so that checks out.